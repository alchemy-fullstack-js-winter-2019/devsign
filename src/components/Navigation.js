import React from 'react';
import { Link } from 'react-router-dom';
import styles from '../components/css/Navigation.css';

export default function Navigation() {
  return (
    <nav className={styles.nav}>
      <ul>
        <Link to="/home"><li>Home</li></Link>
        <Link to="/profile"><li>Profile</li></Link>
        <Link to="/twitter"><li>My Twitter Page</li></Link>
      </ul>
    </nav>
  );
}
