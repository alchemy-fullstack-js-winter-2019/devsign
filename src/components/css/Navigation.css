@import url('https://fonts.googleapis.com/css?family=Margarine');
.nav {
  color: #4c5270;
  img {
    width: 5%;
  }
  ul {
    display: flex;
    flex-flow: row-wrap;
    justify-content: flex-end;
    list-style-type: none;
    padding: 0;
    font-family: 'Margarine', cursive;
  }
 
  a {
    background-color: #4cd7d0;
    text-decoration: none;
    flex: 1 1 auto;

    margin: 10px;
    padding: 1px;
    text-align: center;
    text-transform: uppercase;
    transition: 0.5s;
    background-size: 200% auto;
    color: #4c5270;
    box-shadow: 0 0 20px #4c5270;
    border-radius: 10px;

    :hover {
      background-position: right center; 
      background-image: linear-gradient(to right, #f6d365 0%, #fda085 51%, #f6d365 100%);
    }
  
  }
}