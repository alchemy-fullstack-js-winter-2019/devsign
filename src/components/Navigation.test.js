import React from 'react';
import Navigation from './Navigation';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

describe('Navigation', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <Navigation />
      </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
