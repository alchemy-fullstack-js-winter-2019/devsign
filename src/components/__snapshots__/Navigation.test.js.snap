// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation matches a snapshot 1`] = `
<nav
  className="nav"
>
  <ul>
    <a
      href="/home"
      onClick={[Function]}
    >
      <li>
        Home
      </li>
    </a>
    <a
      href="/profile"
      onClick={[Function]}
    >
      <li>
        Profile
      </li>
    </a>
    <a
      href="/twitter"
      onClick={[Function]}
    >
      <li>
        My Twitter Page
      </li>
    </a>
  </ul>
</nav>
`;
